force-app/classes/src/Assertions.cls

```diff
@@ -6,76 +6,137 @@
  */
 @IsTest
 public class Assertions {
-  public virtual class SystemAsserter {
-    public virtual void assertEquals(
-      Object expected,
-      Object actual,
-      String message
-    ) {
-      System.assertEquals(expected, actual, message);
+  public interface MethodSpyAssertions {
```

Comment:
**praises:** nice abstraction

It helps with the understanding of this class
It will be helpful later, I don't know how nor why but I can feel it :)